sdk/storage/azure-storage-files-datalake/src/datalake_path_client.cpp

```diff
@@ -395,6 +412,18 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
       throw std::runtime_error("Got null value returned when getting access control.");
     }
     ret.Acls = std::move(acl.GetValue());
+    if (result->Owner.HasValue())
+    {
+      ret.Owner = result->Owner.GetValue();
+    }
```

Comment:
I don't understand, if this is not nullable in convenience layer, why is it nullable in protocol layer?

Reply:
This convenience API is calling a protocol layer API shared by several convenience APIs. Please take a look at: https://docs.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/update

For some actions, owner is always in response if it exists.